vendor/github.com/bitrise-steplib/bitrise-step-build-router-start/bitrise/bitrise.go

```diff
@@ -355,6 +359,58 @@ func (artifact BuildArtifact) DownloadArtifact(filepath string) error {
 	return err
 }
 
+// AbortBuild ...
+func (app App) AbortBuild(buildSlug string, abortReason string) error {
+	params := make(map[string]interface{})
+	params["abort_reason"] = abortReason
```

Comment:
This does not show up as the abort reason when the abort request is set.

Any ideas why this might be the case?